tests/playwright/pages/wp-admin-page.js

```diff
@@ -73,15 +74,67 @@ module.exports = class WpAdminPage extends BasePage {
 	}
 
 	/**
-	 * Determine which experiments are active / inactive.
+	 * Activate / Deactivate Experiments.
 	 *
 	 * TODO: The testing environment isn't clean between tests - Use with caution!
 	 *
 	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 * @param {string} mode        - cli (default) or ui
 	 *
 	 * @return {Promise<void>}
 	 */
-	async setExperiments( experiments = {} ) {
+	async setExperiments( experiments = {}, mode = 'cli' ) {
+		if ( 'ui' === mode.toLowerCase() ) {
+			await this._setExperimentsInUI( experiments );
+			return;
+		}
+
+		await this._setExperimentsInCLI( experiments );
+	}
+
+	/**
+	 * Set experiments using the CLI.
+	 *
+	 * @param {Object} experiments - Experiments settings ( `{ experiment_id: true / false }` );
+	 */
+	async _setExperimentsInCLI( experiments = {} ) {
+		let experimentsToEnable = [],
```

Comment:
you are not setting these variables explicitly. I guess they should come from parseExperiments()